File: app/src/main/java/org/medhis2yes/usescases/datasets/datasetDetail/DataSetDetailComponent.java
package org.medhis2yes.usescases.datasets.datasetDetail;

import org.medhis2yes.data.dagger.PerActivity;

import dagger.Subcomponent;


@Subcomponent (modules = DataSetDetailModule.class)
@PerActivity
public interface DataSetDetailComponent {
    void inject(DataSetDetailActivity activity);
}

File: app/src/main/java/org/medhis2yes/usescases/datasets/dataSetTable/dataSetDetail/DataSetDetailComponent.kt
package org.medhis2yes.usescases.datasets.dataSetTable.dataSetDetail

import dagger.Subcomponent
import org.medhis2yes.data.dagger.PerFragment

@Subcomponent(modules = [DataSetDetailModule::class])
@PerFragment
interface DataSetDetailComponent {
    fun inject(dataSetDetailFragment: DataSetDetailFragment)
}
